build.sbt

```diff
@@ -18,3 +18,5 @@ addSbtPlugin("com.geirsson"       % "sbt-scalafmt" % "1.6.0-RC3")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh"      % "0.3.3")
 
 scalafmtOnCompile := true
+
+wartremoverWarnings in (Compile, compile) ++= Warts.unsafe
```

Comment:
This adds settings for the wart remover configuration of the plugin itself.
I think we want to add settings to projects using the plugin via the `projectSettings` method of `ScalazPlugin`